ddtrace/encoding.py

```diff
@@ -14,6 +14,7 @@
     MSGPACK_PARAMS = { 'use_bin_type': True } if version >= (0, 4, 0) else {}
     MSGPACK_ENCODING = True
 except ImportError:
+    MSGPACK_PARAMS = {}
```

Comment:
It seems a Pylons code-path can reproduce a bug where the `MSGPACK_PARAMS` is not set in this branch (so msgpack is disabled but still used).